data_steward/analytics/rt_cdr_qc/cdr_deid_qa_report11_covid_concept_no_suppression.py

```diff
@@ -7,9 +7,9 @@
 #       format_version: '1.5'
 #       jupytext_version: 1.7.1
 #   kernelspec:
-#     display_name: 'Python 3.7.12 64-bit (''.venv'': venv)'
+#     display_name: Python 3
 #     language: python
-#     name: python3712jvsc74a57bd0bd48e0bf57cdd6803c27e3ca6c55ba8b4bab4d98ca7d312da29d41465508be2c
+#     name: python3
```

Comment:
Thanks for updating this. A different notebook had the same issue and I fixed it but missed this one.